src/Microsoft.AspNet.Mvc.Core/Rendering/StringCollectionTextWriter.cs

```diff
@@ -22,6 +22,15 @@ public class StringCollectionTextWriter : TextWriter
         private readonly Encoding _encoding;
 
         /// <summary>
+        /// Creates a new instance of <see cref="StringCollectionTextWriter"/> with <see cref="Encoding"/>
+        /// defaulting to <see cref="UnicodeEncoding"/>.
+        /// </summary>
+        public StringCollectionTextWriter()
```

Comment:
Because this can be used as a replacement for `StringWriter` wherever needed. (I am going to do this in the next PR.) Do you want me to mark this `internal` now and make this `public` in the next one?